src/templates/admin/review/add_review_assignment.html

```diff
@@ -30,9 +30,9 @@ <h2>1. Select Reviewer</h2>
                         {% endblocktrans %}
                     </p>
 
-                    {% if past_reviewers %}
+                    {% if journal_settings.general.display_past_reviewers %}
                         <div class="callout primary">
-                            <p><span class="fa fa-info-circle"></span> Reviewers who have compeleted a review for this article in a previous round will appear at the top of this list, they will show with a <strong>Yes</strong> in the <strong>Has Reviewed Article</strong> column.</p>
+                            <p><span class="fa fa-info-circle"></span> Reviewers who have completed a review for this article in a previous round will appear at the top of this list, they will show with a <strong>Yes</strong> in the <strong>Has Reviewed Article</strong> column.</p>
```

Comment:
Do we even want this setting? It sounds that when it is off, all we do is remove the column and avoid the initial sorting. To me the name of the setting reads as if you would "hide" the past reviewers if I turned this off.